test/e2e/page-objects/pages/settings/shield/shield-claim-page.ts

```diff
@@ -4,15 +4,23 @@ import { Driver } from '../../../../webdriver/driver';
 export default class ShieldClaimPage {
   private readonly driver: Driver;
 
-  // Page identification
   private readonly pageContainer = '[data-testid="submit-claim-page"]';
 
-  // Locators (alphabetically ordered)
   private readonly claimSuccessToast = {
     text: 'Claim submitted successfully',
     tag: 'p',
   };
 
+  private readonly accountSelectorButton =
+    '[data-testid="account-selector-button"]';
+
+  private readonly accountSelectorItem = '.account-selector-modal__account';
+
+  private readonly accountSelectorItemByName = (accountName: string) => ({
+    css: this.accountSelectorItem,
+    text: accountName,
+  });
+
```

Comment:
not a blocker, but it might be good to have the selectors ordered alphabetically.
I'll add a lint rule for that, so we avoid this in the future